integration_test/tests/power_bi_tests.rs

```diff
@@ -434,7 +440,11 @@ mod integration {
             query.push(0);
             assert_eq!(
                 SqlReturn::SUCCESS,
-                SQLExecDirectW(stmt as HStmt, query.as_ptr(), SQL_NTS as SmallInt as i32),
+                SQLExecDirectW(
+                    stmt as HStmt,
+                    query.as_ptr(),
+                    i32::from(SQL_NTS as SmallInt)
```

Comment:
`SQL_NTS` is another example. Every usage of `SQL_NTS` in the code casts it as an `i32`, so we might as well just change the type of the constant and avoid casting entirely.